release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -77,6 +77,10 @@ The {product-title} installation program for Google Cloud Platform (GCP) now cre
 [id="ocp-4-8-web-console"]
 === Web console
 
+[id="ocp-4-8-custom-console-routes-use-custom-domains-cluster-api"]
+==== Custom console routes now use the new `CustomDomains` cluster API
+
+For `console` and `downloads` routes, custom routes functionally is now implemented to use the new `ingress` config route configuration API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent.
```

Comment:
maybe mention that the console-operator method is being deprecated?
```suggestion
For `console` and `downloads` routes, custom routes functionally is now implemented to use the new `ingress` config route configuration API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, since the route configuration via `console-operator` config is being deprecated.
```

Reply:
Thanks! Updated.